.github/workflows/code_review_functions.yml

```diff
@@ -22,18 +22,14 @@ jobs:
       packages: read
       id-token: write
     steps:
-      - uses: actions/checkout@v4
-        with:
-          sparse-checkout: |
-            apps/onboarding-functions
-            libs/
-            test-coverage/
-
-      - name: Set up JDK 17
-        uses: actions/setup-java@v3
+      - name: Setup Maven Action
+        uses: s4u/setup-maven-action@v1.11.0
```

Comment:
you must use hashes instead of version for all third party github actions, see https://pagopa.atlassian.net/wiki/spaces/Technology/pages/734528013/Sicurezza+della+software+supply+chain#GitHub-Actions

Reply:
you can use this tool https://www.npmjs.com/package/pin-github-action